modules/.releng/apps/commerce/app.properties

```diff
@@ -1,4 +1,4 @@
 app.marketplace.id=123396799
 app.marketplace.title=Liferay CE Commerce
 app.marketplace.version=2.0.7
-app.portal.build=7102
\ No newline at end of file
+app.portal.build=7310
```

Comment:
@dsitu this is for the CE app, so it needs to be 7305 as mentioned in my [previous comment](https://github.com/liferay-commerce/liferay-portal/pull/2#issuecomment-675668813)